AirCasting/ABConnector/SDCardAirBeamServices.swift

```diff
@@ -94,8 +87,9 @@ class BluetoothSDCardAirBeamServices: SDCardAirBeamServices {
                     self.finishSync { completion(.failure(SDCardSyncError.wrongOrderOfReceivedPayload)) }
                     return
                 }
-                progress(.chunk(SDCardDataChunk(payload: payload, sessionType: sessionType)))
-                
+                receivedMeasurementsCount[sessionType, default: 0] += 1
+                let progressFraction = Double(receivedMeasurementsCount[sessionType]!) / Double(expectedMeasurementsCount[sessionType].orOne)
```

Comment:
Hmmm... Shouldn't we check here if `expectedMeasurementsCount[sessionType] != 0 ` and if it is, then log an error? Cause if we didn't expect to receive measurements for specific session type, but we did, then we should be aware of this issue.

Reply:
Additionally, this is not exactly what we want to show in the view. We want to show eg. "1/10000" so basically what we have here but as a string and not the value of calculation, cause now it would look like: "mobile: 0.8794234832" etc.